api/Src/Repositories/TeamProjectRepository.ts

```diff
@@ -3,6 +3,10 @@ import * as mongoose from 'mongoose'
 import { TeamProject } from "../Models/TeamProject";
 
 export default class TeamProjectRepository extends Repository {
+    async getAll(){
+      return this.model.find({}).populate('teamId').populate('parentProjectId');
+    }
+
```

Comment:
I changed the backend a bit as otherwise I would have to make more calls from the frontend to get all the required data - prepopulating it made it a bit quicker